tp_chat/handler.py

```diff
@@ -0,0 +1,13 @@
+import logging
+
+from tp_chat.extensions import socket
+
+from flask_socketio import send
+
+@socket.on("message")
+def handle_message(message):
+    logging.info(f"Recived message: {message}")
+    if message == "User connected.":
```

Comment:
Why include this specific if statement? If its related to `socket.send("User {{username}} connected.");` then it is basically always false still (due to username != "").

Reply:
done